<?xml version="1.0"?>
<launch>
   <arg name="mov_filename" default="$(env HOME)/vid.mov" doc="Location of mov_file"/>
   <arg name="display" default="true" doc="To display images or not"/>

   <arg name="left_image" default="camera_image_left" doc="Left image topic"/>
   <arg name="right_image" default="camera_image_right" doc="Right image topic"/>
   <arg name="sonar_image" default="sonar_msg" doc="Sonar message topic"/>

   <arg name="name" default="serdp_player_ros" doc="Serdp_player name"/>

  <node name="serdp_player_ros" pkg="serdp_player" type="$(arg name)">
    <param name="mov_filename" type="string" value="$(arg mov_filename)" />
    <param name="display" type="bool" value="$(arg display)" />

    <remap from="$(arg name)/camera_image_left" to="$(arg left_image)" />
    <remap from="$(arg name)/camera_image_right" to="$(arg right_image)" />
    <remap from="$(arg name)/sonar_image" to="$(arg sonar_image)" />
  </node>

</launch>
